### ISSUE TITLE
Any new version plan?

### ISSUE THREAD

**Issue by JonyTony** [suharev7/clickhouse-rs#187]
Is there a new version plan?
There are some known feat needs.
https://crates.io/crates/clickhouse-rs/1.0.0-alpha.1
https://github.com/suharev7/clickhouse-rs/pull/141

**Comment by suharev7** [comment]
Yes, you right. I didn't release new versions for a long time. Try to release this month.

**Comment by JonyTony** [comment]
邮件以收到，谢谢！  唐春江

**Comment by suharev7** [comment]
Version [1.1.0-alpha.1](https://crates.io/crates/clickhouse-rs/1.1.0-alpha.1) has been released.

**Comment by joshg-ec** [comment]
Is `1.1.0` ready to be stabilized?
